network/src/validation.rs

```diff
@@ -219,7 +211,7 @@ impl<P, E, T> ParachainNetwork for ValidationNetwork<P, E, T> where
 						ready(())
 					});
 
-				let work = select(work, exit).map(drop);
+				let work = work.map(drop);
```

Comment:
@cecton You want to leak all `SharedTable` instances here until the node shuts down?

Reply:
(fixed on last commit, sorry for that! mistakes were made)